src/Backend/Modules/Tags/Engine/Model.php

```diff
@@ -57,12 +52,7 @@ public static function delete($ids)
      */
     public static function exists($id)
```

Comment:
I don't think. I'd remove this, or at least add `@deprecated` in the method signature